input/fsh/mhd.fsh

```diff
@@ -139,6 +139,7 @@ be the the one to use in ITI-68 transactions to retrieve the document content."
 * context.related MS
 * context.related ^comment = "May be filled with URL pointers to Resources or Identifiers found in referenceIdList"
 * context.related.identifier MS
+* context.related.identifier only DicomStudyInstanceUidIdentifier
```

Comment:
Is it the intent to only allow the StudyInstanceUID to be mapped from the `DocumentEntry.referenceIdList`? Such a restriction is not explicit on the XDS side.
Otherwise `context.related` should be sliced.